wpilibc/src/generated/main/native/include/frc/motorcontrol/SD540.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+// THIS FILE WAS AUTO-GENERATED BY ./wpilibc/generate_pwm_motor_controllers.py. DO NOT MODIFY
+
+#pragma once
+
+#include "frc/motorcontrol/PWMMotorController.h"
+
+namespace frc {
+
+/**
+ * Mindsensors SD540 Motor Controller with PWM control.
+ *
+ * Note that the SD540 uses the following bounds for PWM values. These
+ * values should work reasonably well for most controllers, but if users
+ * experience issues such as asymmetric behavior around the deadband or
+ * inability to saturate the controller in either direction, calibration is
+ * recommended. The calibration procedure can be found in the SD540 User
+ * Manual available from Mindsensors.
+ *
+ * \li 2.050ms = full "forward"
+ * \li 1.550ms = the "high end" of the deadband range
+ * \li 1.500ms = center of the deadband range (off)
+ * \li 1.440ms = the "low end" of the deadband range
+ * \li 0.940ms = full "reverse"
```

Comment:
Are the numbers accurate to the thousandths place? (It's not the biggest deal since using 3 decimal places does make the jinja a lot simpler, just double checking)

Reply:
Adding an extra zero won't impact the resulting assembly, so it's really a question of whether we want to claim the extra precision. It's not like anyone is doing propagation of error analysis with these, so it's probably fine.

Reply:
The FPGA uses microseconds, so three digits is appropriate.